crates/bindings-csharp/Runtime/Internal/Module.cs

```diff
@@ -41,20 +43,20 @@ public partial struct ConstraintDef(string name, ColumnAttrs kind, uint[] column
     public partial struct SequenceDef(
         string sequenceName,
         uint colPos,
-        Int128? increment = null,
-        Int128? start = null,
-        Int128? min_value = null,
-        Int128? max_value = null,
-        Int128? allocated = null
+        I128? increment = null,
```

Comment:
This shouldn't be necessary as module runtime only targets latest .NET. Let's keep native Int128 here instead rather than the shim.